src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/VideoLinkBooking.kt

```diff
@@ -33,4 +33,10 @@ data class VideoLinkBooking(
   fun toAppointments(): List<VideoLinkAppointment> {
     return listOfNotNull(pre, main, post)
   }
+
+  fun matchesCourt(court: String?, courtId: String?): Boolean {
```

Comment:
I think it is worth considering making a new class to abstract the court and courtId properties. This method and chooseCourtName() would both be in that class. I think it would help maintainability going forward.

Reply:
It is certainly worth considering.  I expect this PR to be the first of a series of refactorings and feature changes. Subsequent changes will be introducing an 'enabled_courts' table (might not be that name). It will have court_name, court_id pairs. At some point we will only store court id against a video link booking.  Search will then be by id only.  That could be some time though because it requires some sort of change in PSH to handle a list of about 680 court names / ids. 
So plenty of opportunities for improvments / refactorings as the structure becomes clearer.